### ISSUE TITLE
The 'rewards/accuracy_reward' has hardly increased.

### ISSUE THREAD

**Issue by dzyanshan** [om-ai-lab/VLM-R1#163]
I am training with my own data. it is decoded according to the correspondence of letters in the image. However, when I use the following script for training, I find that the 'rewards/accuracy_reward' hardly changes, remaining around 0.3. Could you please tell me why this is happening?
My script:
CUDA_VISIBLE_DEVICES=1 nohup  torchrun --nproc_per_node="1" \
    --nnodes="1" \
    --node_rank="0" \
    --master_addr="127.0.0.1" \
    --master_port="12345" \
    src/open_r1/grpo_jsonl.py \
    --output_dir output/0306 \
    --model_name_or_path /data/home/gpr/code/vlm/Models/Qwen2.5-VL-3B-Instruct \
    --deepspeed local_scripts/zero3.json \
    --dataset_name mydataset \
    --data_file_paths mydataset/data.jsonl \
    --image_folders mydataset/images \
    --max_prompt_length 1024 \
    --num_generations 8 \
    --per_device_train_batch_size 8 \
    --gradient_accumulation_steps 2 \
    --logging_steps 1 \
    --bf16 \
    --torch_dtype bfloat16 \
    --data_seed 42 \
    --report_to wandb \
    --gradient_checkpointing true \
    --attn_implementation flash_attention_2 \
    --num_train_epochs 2 \
    --save_steps 100 \
    --save_only_model true \
    --learning_rate 1e-5 \
    --lora_r 8 \
    --lora_alpha 128 \
    --lora_dropout 0.05 \
    --lora_task_type CAUSAL_LM \
    --freeze_vision_modules true > train_output.log 2>&1 &
My own data is like this:
{"id": "1", "image": "test_1.png", "conversations": [{"from": "human", "value": "<image>Use the decoder in the image to decode an encoded message. The decoded message will be one or more words. When decoding, underscore characters should be mapped to spaces. Show your thought process using <think></think> tags and return the answer within <answer></answer> tags. You can take as much time as you need to think about the problem. While thinking, you should thoroughly verify your solution. Once you have finished thinking, provide your answer in the <answer> section, for example: <answer> cat dog </answer>.\nThe encoded message is: g f d e e h _ f x h h d a"}, {"from": "gpt", "value": "asleep supply"}]}

**Comment by Freddie1946** [comment]
You may need to check your reward function, debug logs could help u
